Sources/Core/Shared/Logging.swift

```diff
@@ -284,11 +310,50 @@ public class LogManager: LogManagerType {
     }
 
     let queue = Queue.Utility.serial("me.danthorpe.Operations.Logger")
-    var enabled: Bool = true
-    var severity: LogSeverity = .Warning
-    var logger: LoggerBlockType = { message, severity, file, function, line in
-        print("\(LogManager.metadataForFile(file, function: function, line: line))\(message)")
+    var enabled: Bool {
+        get { return _enabled.read { $0 } }
+        set {
+            _enabled.write { (value) in
+                value = newValue
+            }
```

Comment:
I'm thinking that it would probably make sense to add subscript support to `Protector` to avoid having to this stuff - quite a bit of boiler plate required when using `Protector` like this...